classes/event/element_updated.php

```diff
@@ -87,6 +87,26 @@ public static function create_from_element(\mod_customcert\element $element) : e
         return self::create($data);
     }
 
+    /**
+     * Create instance of event for the specified element.
+     *
+     * @param int $elementid ID of the element.
+     * @param int $templateid ID of template containing the above element.
+     * @return element_updated
+     */
+    public static function create_from_ids(int $elementid, int $templateid) : element_updated {
```

Comment:
To conform to PSR-12 standards can you please set this to `int $templateid): element_updated` (notice no spacing after the method declaration). Thanks!